### PR TITLE
test: Cypress - Fix flaky test

### PR DESCRIPTION
**Pull request by Parthvi12**
## Description
This PR  fixes flaky tests: RepoLimit and ExportApplication

#### How Has This Been Tested?

locally

#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by Aishwarya-U-R — COMMENTED** [review]
@Parthvi12 - Please send this for review again once test passes consistely.

**Comment by Aishwarya-U-R** [comment]
And which bug are we talking about here?
`[Do not merge until the bug is resolved on EE]
`
Bug id please?

**Comment by Parthvi12** [comment]
> And which bug are we talking about here? `[Do not merge until the bug is resolved on EE] ` Bug id please?

Hey, there was a bug related to repo limit but it was not logged, hence dont have bugid

**Comment by Parthvi12** [comment]
@Aishwarya-U-R For repo limit check I'll raise a separate PR in EE, adding new test. Can we mege this for flaky test fix?

**Comment by Aishwarya-U-R** [comment]
Update title & description of PR.

File: app/client/cypress/e2e/Regression/ClientSide/Git/GitSync/RepoLimitExceededErrorModal_spec.js [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
-    _.onboarding.closeIntroModal();
+    agHelper.AssertElementVisibility(locators._sidebar);
+    onboarding.closeIntroModal();
   });
 
-  it("1. Modal should be opened with proper components", function () {
-    _.homePage.NavigateToHome();
-    _.homePage.CreateNewApplication();
-    _.gitSync.CreateNConnectToGit(repoName1, true, true);
+  it("1. Verify Repo limit flow for CE and EE instances", function () {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Git/GitSync/RepoLimitExceededErrorModal_spec.js:23]: If this is not validating repo limit, can we update title to denote its original validation?

File: app/client/cypress/limited-tests.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # To run only limited tests - give the spec names in below format:
-cypress/e2e/Regression/ClientSide/Widgets/Others/IconButton_2_spec.ts
 
+cypress/e2e/Regression/ClientSide/OtherUIFeatures/ExportApplication_spec.js
```
Comment (Aishwarya-U-R) [inline app/client/cypress/limited-tests.txt:3]: Revert this file changes.

**Comment by Aishwarya-U-R** [comment]
@Parthvi12 - Cypress is prefixed. Please follow this in all future PR's title's.

**Pull request merged**